osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -95,18 +95,24 @@ public abstract class ScoreProcessor
         /// </summary>
         protected virtual bool DefaultFailCondition => Health.Value == Health.MinValue;
 
+        /// <summary>
+        /// Used by specific mods to adjust <see cref="Rank"/>.
+        /// </summary>
+        public BindableBool AdjustRank = new BindableBool();
```

Comment:
I'm not sure I understand why this is a bindable. It does not seem like a good solution.

Reply:
Hmm

Reply:
I only did it for checking if the value was changed, but i came up with a solution, using a property instead, so when the boolean is set, the rank will be recalculated
```c#
private bool adjustRank = false;
public bool AdjustRank
{
    get { return adjustRank; }
    set { adjustRank = value; Rank.Value = rankFrom(Accuracy.Value); };
}
```